types/adone/glosses/std.d.ts

```diff
@@ -33,6 +32,8 @@ import timers = require("timers");
 import dgram = require("dgram");
 import perf_hooks = require("perf_hooks");
 
+declare const assert: any;
```

Comment:
The only dependent for this package on npm is [bloater](https://www.npmjs.com/package/bloater), which just depends on the first 1000 dependencies on npm, and is an attempt to just publish a really large package by the looks of it.

The `adone` npm package itself is deprecated, and the github project, while recently updated ~20 days ago, doesn't seem to have the `std` library anymore.

Of the two maintainers of these types, one no longer exists on github, and the other has no active contributions.

I believe it's acceptable in this case to slightly weaken the types of a single package that doesn't appear to be used anymore, vs not having assertion guards in `@types/node`.

Happy to break this out into another PR if needed.